lib/usd/utils/MergePrims.cpp

```diff
@@ -173,7 +173,8 @@ template <class T> auto makeFuncWithContext(const MergeContext& ctx, T&& func)
 bool isTransformProperty(const UsdProperty& prop)
 {
     static const TfToken xformOpOrderToken("xformOpOrder");
-    return UsdGeomXformOp::IsXformOp(prop.GetName()) || prop.GetBaseName() == xformOpOrderToken;
+    return prop.IsValid()
```

Comment:
This fixes is also unrelated but was causing exceptions during some tests in debug builds only when MAYAUSD_SHOW_FULL_DIAGNOSTICS is on, because USD would throw exception instead of merely logging a message.

Reply:
Not sure about this one, xformOpOrderToken is not used anymore and the function is not testing if the prop is an xfrom but just returning if it is valid. Is this intended?